states/vermont/elections.toml

```diff
@@ -8,7 +8,55 @@ date = 2020-08-11
 name = "Congressional Primary"
 original_date = 2020-08-11
 
+[20200811.registration]
+in_person_by = 2020-08-11T19:00:00
+# online earlier (ensure name appears on checklist)
+# but can register up to election day
+online_by = 2020-08-07
+sources = ["https://sos.vermont.gov/elections/voters/registration/", "https://olvr.vermont.gov"]
+
+[20200811.poll]
+in_person_start = 2020-08-11
+in_person_by = 2020-08-11
+sources = ["https://sos.vermont.gov/elections-calendar/"]
+
+[20200811.poll.early]
+in_person_start = 2020-06-27
+in_person_by = 2020-08-11T19:00:00
+sources = ["https://www.ncsl.org/research/elections-and-campaigns/early-voting-in-state-elections.aspx"]
+
+[20200811.absentee]
+in_person_by = 2020-08-11T19:00:00
+received_by = 2020-08-11T19:00:00
```

Comment:
I think received_by should be the 10th because of "All ballots must be returned to the town clerk’s office before the close of the office on the day before the election"